ruby/2.5.3/Dockerfile

```diff
@@ -11,8 +11,8 @@ RUN mkdir -p /usr/local/etc \
 ENV RUBY_MAJOR 2.5
 ENV RUBY_VERSION 2.5.3
 ENV RUBY_DOWNLOAD_SHA256 1cc9d0359a8ea35fc6111ec830d12e60168f3b9b305a3c2578357d360fcf306f
-ENV RUBYGEMS_VERSION 2.7.6
-ENV BUNDLER_VERSION 1.17.1
+ENV RUBYGEMS_VERSION 3.0.1
```

Comment:
Should this also be `3.0.2` as we're using in Ruby 2.5.0?

Reply:
hm must have fat-fingered that one. thanks.

Reply:
fixed